projects/elements/src/lib/lazy-elements/lazy-elements.module.ts

```diff
@@ -47,6 +62,11 @@ export class LazyElementsModule {
           useValue:
             options && options.elementConfigs ? options.elementConfigs : [],
           multi: true
+        },
+        {
+          provide: ANALYZE_FOR_ENTRY_COMPONENTS,
```

Comment:
Does this find components also if they are in other than `component:` property ?

Reply:
yes